docker/importer/importer.py

```diff
@@ -452,6 +453,63 @@ def thread_init():
 
     logging.info("Finished processing bucket: %s", source_repo.name)
 
+  def _process_updates_rest(self, source_repo: osv.SourceRepository):
+    """Process updates from REST API."""
+    logging.info("Begin processing REST: %s", source_repo.name)
+    import_time_now = utcnow()
+    request = requests.head(source_repo.rest_api_url, timeout=60)
+    if request.status_code != 200:
+      logging.error('Failed to fetch REST API: %s', request.status_code)
+      return
+    last_modified = datetime.datetime.strptime(request.headers['Last-Modified'],
+                                               '%a, %d %b %Y %H:%M:%S %Z')
+    # Check whether endpoint has been modified since last update
+    if last_modified < source_repo.last_update_date:
+      logging.info('No changes since last update.')
+      return
+    request = requests.get(source_repo.rest_api_url, timeout=60)
+    # Get all vulnerabilities from the REST API. (CURL approach)
+    vulns = request.json()
+    vulns_to_update = []
+    # Create tasks for changed files.
+    for vuln in vulns:
+      import_failure_logs = []
+      last_modified = datetime.datetime.strptime(vuln['modified'],
+                                                 '%Y-%m-%dT%H:%M:%S.%fZ')
+
+      if last_modified < source_repo.last_update_date:
+        continue
+      try:
+        if source_repo.link[-1] != '/':
```

Comment:
nit: can we just enforce that these should always end with '/' by throwing an exception at the start of all of this if this is not the case?

I think we already make this assumption everywhere else